docs/intro/reusable-apps.txt

```diff
@@ -262,6 +262,26 @@ this. For a small app like polls, this process isn't too difficult.
    you add some files to it. Many Django apps also provide their documentation
    online through sites like `readthedocs.org <https://readthedocs.org>`_.
 
+   Many Python projects, including Django and Python itself, use `Sphinx
+   <https://www.sphinx-doc.org/en/master/usage/quickstart.html>`_ to build
+   their documentation. If you choose to use Sphinx you can link back to the
+   Django documentation by configuring `Intersphinx
+   <https://www.sphinx-doc.org/en/master/usage/quickstart.html#intersphinx>`_
+   and including a value for Django in your project's ``intersphinx_mapping``
+   value::
+
+        intersphinx_mapping = {
+            # ...
+            "django": (
+                "https://docs.djangoproject.com/en/stable/",
+                "https://docs.djangoproject.com/en/stable/_objects/",
```

Comment:
I've merged https://github.com/django/djangoproject.com/pull/1995, so we can now use the default location of `objects.inv`:
```suggestion
                None,
```

Reply:
Great. Yes. 🙌